docs/OncoTree-Mapping-Tool.md

```diff
@@ -30,163 +30,163 @@ The list of OncoTree versions available are viewable [here](http://oncotree.mskc
 
-**The `ONCOTREE_SUGGESTIONS` column would be shown as follows:** 
-> _TALL -> {TLL}, more granular choices introduced_  
+ > _'Source Code' -> ???, Oncotee code not in source oncotree version_
 
-### 5. Invalid Source OncoTree Code  
- An invalid souce OncoTree Code means the provided code cannot be found in the source version. In such a case, mapping cannot be attempted and the `ONCOTREE_SUGGESTSIONS` column displays the following:  
- 
- > _'Source Code' -> ???, Oncotee code not in source oncotree version_ 
- 
 ### [Tutorial](#tutorial)
-The following tutorial will guide the user through using the **oncotree_to_oncotree.py** tool. The tutorial will go through the expected output to highlight specific mapping cases. Additionally, the tutorial will cross-reference the output with the generated mapping summary to demonstrate how it can be used to aid in manual selection of unresolved nodes. 
-  
-All files used in this tutorial (both input and expected output) can be found in the [Additional Resources](#additional-resources) section. 
+The following tutorial will guide the user through using the **oncotree_to_oncotree.py** tool. The tutorial will go through the expected output to highlight specific mapping cases. Additionally, the tutorial will cross-reference the output with the generated mapping summary to demonstrate how it can be used to aid in manual selection of unresolved nodes.
 
-#### Step 1 
-Download the sample input file (data_clinical_sample_test.txt) file from the [Additional Resources](#additional-resources) section.
+All files used in this tutorial (both input and expected output) can be found in the [Additional Resources](#additional-resources) section.
+
+#### Step 1
+Download the sample input file (data_clinical_sample.txt) file from the [Additional Resources](#additional-resources) section.
 #### Step 2
-Download oncotree_to_oncotree.py from [here](http://oncotree.mskcc.org/downloads/oncotree_to_oncotree.py).
-#### Step 3 
-Run the following command from the commandline:  
-  
-`python oncotree_to_oncotree.py -i data_clinical_sample_test.txt -o data_clinical_sample_test_remapped.txt -s oncotree_2018_03_01 -t oncotree_2019_03_01`  
-  
-The tool will output two files: `data_clinical_sample_test_remapped.txt` and `data_clinical_sample_remapped_summary.html`.  
+Download oncotree_to_oncotree.py from [here <span class="text-primary oi oi-data-transfer-download"></span>](http://oncotree.mskcc.org/downloads/oncotree_to_oncotree.py).
+#### Step 3
+Run the following command from the commandline:
+
+`python oncotree_to_oncotree.py -i data_clinical_sample.txt -o data_clinical_sample_remapped.txt -s oncotree_2018_03_01 -t oncotree_2019_03_01`
+
+The tool will output two files: `data_clinical_sample_remapped.txt` and `data_clinical_sample_remapped_summary.html`.
 #### Step 4
```

Comment:
Lets drop the last column of all ellipses and just say "Examine data_clinical_sample.remapped.txt, its first five columns should look be as follows (with the remaining columns intact):

Maybe that will give you some breathing space between the displayed columns too.

Reply:
done